bundle/manifests/nic-configuration-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,373 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
```

Comment:
missing `containerImage` with the operator image sha256
See:
https://github.com/redhat-openshift-ecosystem/certified-operators/blob/main/operators/nvidia-network-operator/24.10.0/manifests/nvidia-network-operator.clusterserviceversion.yaml#L5